packages/web-components/src/components/modal/modal.scss

```diff
@@ -0,0 +1,101 @@
+//
+// Copyright IBM Corp. 2020
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '@carbon/ibmdotcom-styles/scss/components/expressive-modal/expressive-modal';
+@import 'carbon-custom-elements/scss/components/modal/modal';
+@import '../../globals/scss/vars';
+
+:host(#{$dds-prefix}-modal) {
+  @extend :host(#{$prefix}-modal);
+  @extend .#{$prefix}--modal--expressive;
+
+  &[open] {
+    @extend :host(#{$prefix}-modal[open]);
+  }
+
+  &[expressive-size='full-width'] {
+    ::slotted(#{$dds-prefix}-modal-body),
+    ::slotted(#{$dds-prefix}-lightbox-media-viewer-body) {
+      padding-right: 0;
+      height: auto;
+    }
+
+    ::slotted(#{$dds-prefix}-modal-body) {
+      min-height: carbon--rem(500px);
+    }
+
+    ::slotted(#{$dds-prefix}-lightbox-media-viewer-body) {
+      height: 100%;
+      overflow: auto;
+    }
+
+    .#{$prefix}--modal-close {
+      top: 0;
+      right: 0;
+    }
+
+    @include carbon--breakpoint('md') {
+      .#{$prefix}--modal-container {
+        width: calc(100% - #{$carbon--spacing-07});
+        max-width: calc(100% - #{$carbon--spacing-07});
+        height: calc(100% - #{$carbon--spacing-07});
+        max-height: none;
+      }
+    }
+  }
+
+  .#{$dds-prefix}-ce--modal__hedaer--with-body {
```

Comment:
@asudoh is this a typo in the classname in the custom elements side?

Reply:
`#{$dds-prefix}-ce` prefix is for telling that such CSS class happens only with custom elements implementation, but I'm open for removing such `-ce` prefix.

Reply:
@asudoh I meant `header` instead of `hedaer`